salt/edx/hacks.sls

```diff
@@ -71,21 +52,5 @@ add_xpro_base_url_to_{{ app }}_production_file:
     - text: XPRO_BASE_URL = '{{ heroku_env }}'
```

Comment:
We should commit this directly to edx-platform and maintain it as a cherry pick rather than appending to the file at deploy time. We'll want to add it to production.py so that it reads the value in from lms.yaml and then add that value in the lms_env_extra in the ansible vars.

Reply:
Does it make sense though to have it be in the edx-platform when it's specific to xpro?

Reply:
It would be on the xpro branch